airflow/providers/google/suite/transfers/gcs_to_gdrive.py

```diff
@@ -63,12 +63,15 @@ class GCSToGoogleDriveOperator(BaseOperator):
         For example, with prefix ``foo/*`` and destination_object ``blah/``, the file ``foo/baz`` will be
         copied to ``blah/baz``; to retain the prefix write the destination_object as e.g. ``blah/foo``, in
         which case the copied file will be named ``blah/foo/baz``.
+    :param destination_folder_id: The folder ID where the destination objects will be placed.  It is
+        an additive prefix for anything specified in destination_object.
+        For example, if folder ID ``xXyYzZ`` is called ``foo``, destination is ``bar/baz`` and source object
+         is ``qux``, the file will end up in `foo/bar/baz`.
```

Comment:
What does the destination object have to do with the source object? If it's irrelevant, mentioning it in the example is a bit confusing

Reply:
@stiak can you please address the above comment?

Reply:
Ok, will remove the reference to source object.